addons/hr_contract/migrations/12.0.1.0/post-migration.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2.extensions import AsIs
+
+
+def add_group_hr_contract_manager(env):
+    hr_manager = env.ref('hr.group_hr_manager').id
+    new_group = env.ref('hr_contract.group_hr_contract_manager').id
```

Comment:
This is simpler to be solved:
```python
env.ref('hr_contract.group_hr_contract_manager').users = [
    (4, x.id) for x in env.ref('hr.group_hr_manager').users
]
```

Reply:
but this way is not logged. the query way logs the change :)

Reply:
Don't worry about that in that case. If you want some logging, just put `openupgrade.logging()` decorator in the method, but having the exact "query" is not critical here. Or if you want to be verbose, put an explicit logging saying `Adding "HR contract manager" permission to users %s...`.

Reply:
ok, replaced. bye bye sexy query 🤧

Reply:
Hehe, sexy but complicated, as in real life :yum: